src/content/code/services/typetta.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Typetta
+description: Typetta is an open-source ORM written in TypeScript that aims to allow seamless access to data in a typed fashion to all main SQL databases (MySQL, PostgreSQL, Microsoft SQL Server, SQLLite3, CockroachDB, MariaDB, Oracle e Amazon Redshift) and also to the NoSQL database MongoDB. For more information, go to https://twinlogix.github.io/typetta/
```

Comment:
```suggestion
description: Typetta is an open-source ORM written in TypeScript that aims to allow seamless access to data in a typed fashion to all main SQL databases (MySQL, PostgreSQL, Microsoft SQL Server, SQLLite3, CockroachDB, MariaDB, Oracle * Amazon Redshift) and also to the NoSQL database MongoDB.
```

Reply:
I've cleaned up the English a bit and removed the redundant link - this should be good to go after that,  thanks for providing more context